### PR TITLE
LPS-161853 | Master 

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-161853

cc: @patriciajeanperez

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 1]
```diff
@@ -32,6 +32,63 @@ definition {
...
+			SelectFrame.selectFrameNoLoading(
+				fieldLabel = "Numeric",
+				locator1 = "DataEngineBuilder#FORM_FIELD_NESTED");
+
+			FormFields.resizeFieldGroup(
+				fieldName = "Numeric",
+				moveForce = "-300");
+		}
+
+		task ("Then all fields in the field group are present") {
```
Comment (kristinonias) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:85]: This macro asserts both the field group and the fields in the field group.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 2]
```diff
@@ -32,6 +32,63 @@ definition {
...
+
+			FormFields.resizeField(
+				fieldName = "Text",
+				moveForce = "-400");
+
+			FormFields.moveFieldToPosition(
+				fieldColumnNumber = 1,
+				fieldLabel = "Date");
+
+			Click(locator1 = "Sidebar#BACK");
```
Comment (kristinonias) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:63]: For some reason, the side bar does not navigate back to the fields, so added this to ensure that the third field can be added.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 3]
```diff
@@ -32,6 +32,63 @@ definition {
...
+		}
+
+		task ("and when another resized field is added to the field group") {
+			DataEngine.addFieldToFieldsGroup(
+				fieldFieldLabel = "Numeric",
```
Comment (kristinonias) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:68]: The test steps originally ask for another text field, however I was unable to resize the second field, so I changed it to numeric as the second text field isn't necessarily and specifically needed for this test to pass.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**